pkg/pureutils/pure.go

```diff
@@ -0,0 +1,26 @@
+package pureutils
+
+import (
+	"github.com/portworx/torpedo/pkg/units"
+	"strings"
+)
+
+// GetPureAppDataDir checks the pod namespace prefix, and returns a path that we can
+// write data to on a volume. Because the mountpath varies so heavily between applications
+// (some have multiple PVCs, some have configmap vols, etc. etc.) this is the easiest way
+// at the moment. As more apps get added to our test suite, we should add them here.
+func GetPureAppDataDir(namespace string) (string, int) {
```

Comment:
```suggestion
func GetAppDataPath(namespace string) (string, int) {
```
Remove Pure. It's not related.